### ISSUE TITLE
Resources

### ISSUE THREAD

**Issue by dariusk** [dariusk/NaNoGenMo-2014#1]
This is an open issue where you can comment and add resources that might come in handy for NaNoGenMo.

There are already a ton of resources on [the old resources thread](https://github.com/dariusk/NaNoGenMo/issues/11) for the 2013 edition.

**Comment by jkervinen** [comment]
Hi, I assume this (NaNoGenMo 2014)  has ended already, was it Dec 1, 2014 the deadline ?

**Comment by hugovk** [comment]
@jkervinen Yep, it ran from 12:01am GMT on Nov 1st 2014 to 12:01am GMT Dec 1st 2014. Welcome back in November 2015.

There's also an out-of-season email group: https://github.com/dariusk/NaNoGenMo-2014/issues/155

**Comment by jkervinen** [comment]
@hugovk Thanks, participating in November 2015, and checking the email group !

**Comment by hugovk** [comment]
Not long until NaNoGenMo 2015! 

And just in time, @samplereality is amongst the instructors of a free, online, six-week course on Electronic Literature, starting today. Sounds perfect for NaNoGenMo inspiration!

> Electronic Literature

> Interactive Fiction. Chatterbots. Hypertext and kinetic poetry. Explore these avant garde forms and other new media in this exciting class.

https://www.edx.org/course/electronic-literature-davidsonx-d004x
